mobile/src/modules/auth/store/actions.ts

```diff
@@ -57,14 +59,13 @@ export const createActiveAccount = createActionFn<ActiveAccountGeneratorData, Ac
     const pinHash = hashSHA256(pin + keys.publicKey);
 
     // TODO: make fields optional in @burst package
-    // @ts-ignore
-    return {
+    return new Account({
```

Comment:
Thing that i strongly disagree: account interface vs account implementation. Interface says, that all fields are required and always presented, but implementation says, that there can be tons of `undefined` values. And we will get a crash, for example, on `account.fieldName.toString()`. And TypeScript with strict mode can't help us

Reply:
This is a problem in the desktop wallet too. For now, I propose we continue to use the Account object so we have a consistent model between the two apps. Eventually, we should circle back and replace the Account class with a simpler Account interface.

Reply:
This account class is bothering me too... legacy code. We should do something.